examples/composite_queries/src/canister2/canister2.py

```diff
@@ -0,0 +1,43 @@
+from kybra import Async, CanisterResult, ic, manual, nat, Principal, query, update
+from src.canister1.types import StringQueryResult
+from src.canister3.types import Canister3
+
+canister3 = Canister3(Principal.from_str('r7inp-6aaaa-aaaaa-aaabq-cai'))
+
+counter: nat = 0
+
+
+@query
+def inc_counter() -> nat:
```

Comment:
This is a composite query in azle but is not here in kybra

Reply:
I'll need some help with this one. Pylance is giving me some weird errors, that I cannot figure out.